main.tf

```diff
@@ -0,0 +1,31 @@
+resource "helm_release" "ingress_nginx_release" {
+  name       = "nginx-ingress"
+  repository = "https://kubernetes.github.io/ingress-nginx"
+  chart      = "ingress-nginx"
+  version    = var.chart_version
+  namespace  = var.namespace
+
+  values = concat(
+    [templatefile(
+    "${path.module}/values.yaml",
+    {
+      ssl_reject_handshake = var.ssl_reject_handshake
+    }
+  )],
```

Comment:
I think that we have an indentation problem here. Same as before, use the formatter to fix this issue.